templates/module/salt/modules/{{module_name}}.py

```diff
@@ -13,29 +13,23 @@
         <your example config>
 
 '''
-# keep lint from choking on _get_conn and _cache_id
-#pylint: disable=E0602
-
-from __future__ import absolute_import
-
 # Import Python libs
+from __future__ import absolute_import
 import logging
 
 # Import salt libs
 import salt.utils.compat
 
-log = logging.getLogger(__name__)
-
 # Import third party libs
-HAS_LIBS = False
 try:
     #  Import libs...
     {% if depending_libraries %}
     import {{depending_libraries}}
     {% endif %}
     HAS_LIBS = True
 except ImportError as ie:
-    missing_package = ie.message
+    HAS_LIBS = False
+    MISSING_PACKAGE_REASON = ie.message
```

Comment:
Fixed locally. Let me known if/when you're done with the review so I can force push the changes.